pyvcloud/vcd/org.py

```diff
@@ -107,6 +107,35 @@ def get_catalog(self, name):
                 return self.client.get_resource(link.href)
         raise Exception('Catalog not found.')
 
+    def update_catalog(self, old_catalog_name, new_catalog_name, description):
+        """
+        Update the name and/or description of a catalog.
+        :param old_catalog_name: (str): The current name of the catalog.
+        :param new_catalog_name: (str): The new name of the catalog.
+        :param description: (str): The new description of the catalog.
+        :return:  A :class:`lxml.objectify.StringElement` object describing the updated catalog.
+        """
+        org = self.client.get_resource(self.href)
```

Comment:
if self.resource is None:
             self.resource = self.client.get_resource(self.href)

Will save an extra ReST call to vCD if Org object is already initialized.